src/OpenLoco/src/World/Company.h

```diff
@@ -163,7 +163,8 @@ namespace OpenLoco
         World::SmallZ headquartersZ;          // 0x2579
         coord_t headquartersX;                // 0x257A -1 on no headquarter placed
         coord_t headquartersY;                // 0x257C
-        uint8_t pad_257E[0x259E - 0x257E];
+        currency32_t var_257E;
```

Comment:
can we name this?

Reply:
it is named in one of the follow on PR's